src/ray/common/ray_config_def.h

```diff
@@ -482,6 +482,9 @@ RAY_CONFIG(uint64_t, gcs_service_address_check_interval_milliseconds, 1000)
 /// Normally each metrics is about < 1KB. 1000 means it is around 1MB.
 RAY_CONFIG(int64_t, metrics_report_batch_size, 1000)
 
+/// If task events (status change and profiling events) from driver should be ignored.
+RAY_CONFIG(bool, task_events_skip_driver, false)
```

Comment:
why do we need this?

Reply:
This is to skip recording for driver in test to have fine-grained control of the tasks. 

I plan to refactor the profiling recording in a follow-up PR (which should remove this).